### DISCUSSION TITLE
Home Assistant su Surface Go

### DISCUSSION THREAD

**Discussion by M4ur1no** [indomus/forum#2194] [HUB: Home Assistant]
Buongiorno a tutti!
Mi sto avvicinando al mondo home assistant, vorrei creare il mio hub personale da dove poter gestire tutte le apparecchiature connesse e compatibili.
Ho un dubbio però che vorrei sciogliere prima di poter iniziare: 
la mia idea è quella di tenere un tablet a parete dal quale gestire tutto l'hub di HA, avendo io a disposizione un surface go che non utilizzo e non avendo ancora comprato nè un minipc nè un raspberry, mi conviene installare home assistant direttamente su questo tablet che comunque resterà sempre alimentato?
Ci sono altri motivi oltre al consumo che dovrebbero farmi propendere per un'installazione su mini pc o raspberry?
Grazie mille a chiunque voglia darmi una mano!

**Comment by snakuzzo** [comment]
dipende da quello che ci devi fare...fai prima a provare direttamente, secondo me

**Reply by M4ur1no** [reply]
Vorrei gestirci la domotica (utilizzo shelly), comprese le telecamere, l'aspirapolvere robot, e l'antifurto (sempre con sensori shelly) tutto in una schermata dinamica da appesa al muro in sostanza.

**Reply by snakuzzo** [reply]
premesso che non conosco le caratteristiche di quel tablet, ma l'unica incognita sono le telecamere.
Il discorso legato ad esse dipende da tanti fattori...in primis "se il server deve fare transcodifica" andando ad impattare conseguentemente sulla CPU. Questo devi saperlo tu, in base anche al tipo di flusso video che hai, da quanti ne "visualizzi" in contemporanea, se fa transcoding anche quando non visualizzi la dashboard, etc.
Inoltre, non è detto che con un qualsiasi mini PC riusciresti ad assorbire eventuale carico CPU, in caso di pesante transcodifica avresti bisogno di una CPU performante e/o di una GPU che ti permetta di scaricare il lavoro lato CPU.
Alla fine io ti direi... prova...poi se e quando dovesse nascere l'esigenza di un hardware più performante si ragione. In quel caso sposti HA su un server e tieni il tablet solo come client con l'app companion o con lo shortcut alla tua istanza locale

**Reply by M4ur1no** [reply]
Non ti so rispondere sulle telecamere, è il primo approccio ad HA e direi che forse è anche un po' precoce parlare di cosa riuscirò ad integrare. 
Penso proprio che farò una prova, come hai detto tu.
Grazie mille intanto!!

**Comment by KC-inDomus** [comment]
> Buongiorno a tutti! Mi sto avvicinando al mondo home assistant, vorrei creare il mio hub personale da dove poter gestire tutte le apparecchiature connesse e compatibili. Ho un dubbio però che vorrei sciogliere prima di poter iniziare: la mia idea è quella di tenere un tablet a parete dal quale gestire tutto l'hub di HA, avendo io a disposizione un surface go che non utilizzo e non avendo ancora comprato nè un minipc nè un raspberry, mi conviene installare home assistant direttamente su questo tablet che comunque resterà sempre alimentato? Ci sono altri motivi oltre al consumo che dovrebbero farmi propendere per un'installazione su mini pc o raspberry? Grazie mille a chiunque voglia darmi una mano!

come dice anche diego, dipende che devi fare. sopratutto, cerca di capire cosa gira sul tablet come OS (personalmente non lo conosco)

**Reply by M4ur1no** [reply]
Farò qualche prova prima di decidere!
Grazie mille
